tests/FSharp.Test.Utilities/ProjectGeneration.fs

```diff
@@ -896,4 +1176,51 @@ type SyntheticProject with
     /// Saves project to disk and checks it with default options. Returns the FSharpChecker that was created
     member this.SaveAndCheck() =
         this.Workflow.Yield() |> Async.RunSynchronously |> ignore
-        this.Workflow.Checker
\ No newline at end of file
+        this.Workflow.Checker
+
+    static member CreateFromRealProject(projectDir) =
```

Comment:
Nowhere in `main` so far. CI should be capable of using this if we provide a local copy of some project (not sure if it can handle anything we have in the repo).

Reply:
I think this folder ~\tests\projects\stress or its parent should have eligible projects.